### ISSUE TITLE
Readability.MultiAlias does not work when using `__MODULE__` as the root of an alias

### ISSUE THREAD

**Issue by jared-mackey** [rrrene/credo#957]
### Environment

* Credo version (`mix credo -v`): `1.6.4`
* Erlang/Elixir version (`elixir -v`): 
```
Erlang/OTP 24 [erts-12.3] [source] [64-bit] [smp:32:32] [ds:32:32:10] [async-threads:1] [jit]

Elixir 1.13.3 (compiled with Erlang/OTP 24)
```
* Operating system: Arch
* Credo strict: yes

### What were you trying to do?
Use the `Readability.MultiAlias` check against code like this:

```elixir
alias __MODULE__.{Thing, OtherThing}
```

### Expected outcome
Credo should should report an issue for that check.

### Actual outcome
Credo does not report an issue.

**Comment by rrrene** [comment]
Oh, that is a bug.

I would definitely merge a PR fixing this!

**Comment by jared-mackey** [comment]
@rrrene No worries. I won’t be able to spend any time on this myself either. Could you label it as help wanted?

**Comment by avinayak** [comment]
I made an attempt to fix this https://github.com/rrrene/credo/pull/970. I hope this is sufficient.
